server/cla.go

```diff
@@ -40,6 +41,10 @@ func (s *Server) checkCLA(pr *model.PullRequest) {
 		return
 	}
 
+	if s.IsAlreadySigned(pr) {
```

Comment:
When does this case happen? New commits overwrite the old status, which means this is never true for sync events.